index.bs

```diff
@@ -308,57 +308,43 @@ See also:
 
 <h3 id="secure-context">Consider limiting new features to secure contexts</h3>
 
-It may save you significant time and effort
-to pre-emptively restrict your feature to Secure Contexts.
-
-The TAG is on the record in supporting
-an industry-wide move to [Secure the Web](https://www.w3.org/2001/tag/doc/web-https)
-and applaud [efforts](https://blog.mozilla.org/security/2017/01/20/communicating-the-dangers-of-non-secure-http/)
-to shift web traffic to secure connections.
-A great deal of effort has gone into debating
-which features should be restricted to [Secure Contexts](https://w3c.github.io/webappsec-secure-contexts/).
-Opinions vary amongst engine vendors, leading to difficult choices for feature designers.
-Some vendors require *all* new features be restricted this way,
-whereas others take a more selective approach.
-
-This backdrop makes it difficult to provide
-advice about the extent to which your feature should be restricted.
-What we *can* highlight is that Secure Context-restricted features
-face the least friction in gaining wide adoption amongst these varying regimes.
+Always limit your feature to secure contexts
+if it would pose a risk to the user
+without the authentication, integrity, or confidentiality
+that's present only in secure contexts.
 
+<div class="example">
+One example of a feature that should be limited to secure contexts
+is [Geolocation](https://www.w3.org/TR/geolocation-API/),
+since it would be a risk to users' privacy to transmit their location in an insecure way.
+</div>
+
+For other features, carefully weigh up whether
+<a href="priority-of-constituencies">user needs</a> are best served
+by making the feature available in an insecure context.
```

Comment:
So the previous text here reflected a lack of TAG consensus on whether we should be asking for:
* all new features to be limited to secure contexts (in order to move the web more quickly to https, which does serve user needs given that many of the existing features of the web are often privacy-sensitive), or
* only limiting features to secure contexts when there's a specific reason that they're *more* sensitive

This text much more concretely takes a position on that question where we didn't previously have consensus (see #32, #75, and #89 for the history).

If I were to try to summarize that disagreement clearly, I'd say (replacing the last sentence here):

> For other features, the TAG does not have consensus.  Some members of the TAG believe that all clearly-distinct new features should be limited to secure contexts.  Others believe that those designing new features should make decisions for each feature based on whether user needs are best served by making the feature available in an insecure context.

Reply:
@dbaron, I suppose I'm one of those others, and I don't think your last sentence represents my position. I believe that bifurcating the web's API surface is harmful, and therefore, only features with privacy or security impact should be limited to secure contexts.

Reply:
I had a go at representing both of these positions.

Reply:
@hober since David is out, could you take a look at the wording here and see if it seems fair?

Reply:
Yes, it seems fair to me.